client/my-sites/plans/current-plan/index.jsx

```diff
@@ -193,7 +195,244 @@ class CurrentPlan extends Component {
 	}
 
 	renderEcommerceTrialPage() {
-		return <div className="current-plan__ecommerce-trial-wrapper">{ this.renderMain() }</div>;
+		const { translate } = this.props;
+
+		return (
+			<>
+				<BodySectionCssClass bodyClass={ [ 'is-trial-plan' ] } />
+
+				<Card className="current-plan__trial-card">
+					<div className="current-plan__trial-card-content">
+						<Badge type="info">{ translate( '5 days left' ) }</Badge>
+						<p className="current-plan__card-title">
+							{ translate( 'You’re in a free trial store' ) }
+						</p>
+						<p className="current-plan__card-subtitle">
+							{ translate(
+								'Your free trial we’ll expire in 5 days. Pick a plan by December, 13th to keep your store running.'
+							) }
+						</p>
+						<Button primary>{ translate( 'Pick a plan' ) }</Button>
+					</div>
+					<div className="current-plan__trial-ilustration">
+						<img
+							width={ 180 }
+							alt="Pick a plan"
+							src="/calypso/images/plans/wpcom/plan-ecommerce-trial.png"
+						/>
+					</div>
+				</Card>
+
+				<h2 className="current-plan__section-title">What’s included in your free trial</h2>
+				<div className="current-plan__included-wrapper">
+					<Card>
+						<img
+							className="current-plan__included-ilustration"
+							alt="Priority support"
+							src="/calypso/images/plans/wpcom/trial-features/priority-support.svg"
+						/>
+						<p className="current-plan__included-title">Priority support</p>
+						<p className="current-plan__included-text">
+							Need help? Reach out to us anytime, anywhere.
+						</p>
+						<Button className="current-plan__included-link">Ask a question</Button>
```

Comment:
Please add `translate()` wrappers to the strings.
```suggestion
							alt={ translate( 'Priority support' ) }
							src="/calypso/images/plans/wpcom/trial-features/priority-support.svg"
						/>
						<p className="current-plan__included-title">{ translate( 'Priority support' ) }</p>
						<p className="current-plan__included-text">
							translate( 'Need help? Reach out to us anytime, anywhere.' )
						</p>
						<Button className="current-plan__included-link">{ translate( 'Ask a question' ) }</Button>
```